### PR TITLE
initial upload of gcp-cli orb - DO NOT MERGE

### PR DESCRIPTION
**Pull request by ttrahan**
Commands, executors, and jobs in this orb have been tested here:
https://circleci.com/workflow-run/77a59cea-30a3-47d1-8048-8090c1ecbd99

### CODE REVIEW

File: src/gcp-cli/orb.yml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      - image: google/cloud-sdk
+
+commands:
+  install:
+    description: |
+      Install the gcloud CLI, if not available
+    steps:
+      - when:
+          name: Install gcloud CLI, if not available
+          condition: $(command -v gcloud) == ""
```
Comment (eddiewebb) [inline src/gcp-cli/orb.yml]: Does this work? I  thought conditionals were evaluated before instantiating any containers.
>These conditions are checked before a workflow is actually run. That means, for example, that a user can't use a condition to check an environment variable.

File: src/gcp-cli/orb.yml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+          steps:
+            - run:
+                name: Install gcloud CLI
+                command: |
+                  # Set sudo to work whether logged in as root user or non-root user
```
Comment (eddiewebb) [inline src/gcp-cli/orb.yml]: I'm stealing this. 💯

**Comment by ttrahan** [comment]
@eddiewebb resolved the incorrect use of when condition. Should be ready to go now.

**Comment by iynere** [comment]
closing this, as it's here now :  https://github.com/CircleCI-Public/gcp-cli-orb

**Pull request closed without merging**